### PR TITLE
Editing Toolkit: Update to v2.20

### PR DESCRIPTION
**Pull request by andrewserong**
#### Changes proposed in this Pull Request

* Bump version of Editing Toolkit plugin to 2.20

## [Editing toolkit changes included in this release](https://github.com/Automattic/wp-calypso/commits/trunk/apps/editing-toolkit)

* [ ] remove negative left margin (https://github.com/Automattic/wp-calypso/pull/50250) @Addison-Stavlo 
* [x] Launch: Fix logic to compute redirectToWpcomPath origin (https://github.com/Automattic/wp-calypso/pull/50241) @ciampo
* [x] Tidy up after recent Focused Launch checkout changes (https://github.com/Automattic/wp-calypso/pull/50182) @ciampo
* [x] i18n: Add missing translators comments in ETK components (https://github.com/Automattic/wp-calypso/pull/50190) @yuliyan 
* [x] Disable premium block patterns feature due to performance issues (https://github.com/Automattic/wp-calypso/pull/50111) @andrewserong 
* [x] Patterns: Allow custom viewport width when registering patterns (https://github.com/Automattic/wp-calypso/pull/49906) @andrewserong 
* [x] Remove Anchor-specific welcome guide (https://github.com/Automattic/wp-calypso/pull/50161) @allilevine 
* [x] Launch: Cleanup calypsoifyGutenberg (https://github.com/Automattic/wp-calypso/pull/49067) @yansern @ciampo 
* [x] Focused Launch: Only launch site if checkout is complete (https://github.com/Automattic/wp-calypso/pull/48452) @StefanNieuwenhuis  @razvanpapadopol 
* [x] chore(@automattic/page-template-modal): Use the same dependencies than ETK (https://github.com/Automattic/wp-calypso/pull/50114) @scinos @roo2 
* [x] Update Newspack Blocks to v1.21.0 (https://github.com/Automattic/wp-calypso/pull/50154) @dkoo 
* [x] Focused Launch - Summary View: Fix incorrect HE support link (https://github.com/Automattic/wp-calypso/pull/50115) @StefanNieuwenhuis 
* [x] Patterns: Reveal the Featured pattern category and make it the first in the list (https://github.com/Automattic/wp-calypso/pull/49907) @andrewserong 
* [x] Move page template modal into its own package (https://github.com/Automattic/wp-calypso/pull/49661)  @roo2

### Other [changes](https://github.com/Automattic/wp-calypso/commits/trunk/packages) included from the [imported @automattic packages](https://github.com/Automattic/wp-calypso/blob/trunk/apps/editing-toolkit/tsconfig.json#L37-L46) and their imports

* [x] Page layout picker: Fix broken web fonts in page layout picker large preview (https://github.com/Automattic/wp-calypso/pull/50253)
* [x] Billing: Hide disallowed payment methods when changing payment method (https://github.com/Automattic/wp-calypso/pull/49959) @sirbrillig (Does this change affect the ETK release? I don't _think_ so, but we might want to double check)
* [x] Focused Launch: Fix billing period tracking in summary (https://github.com/Automattic/wp-calypso/pull/49930) @ciampo
* [x] Focused Launch: Fix domain picker item flex spacing (https://github.com/Automattic/wp-calypso/pull/49883) @ciampo
* [x] plans-grid: Fix translators comment format (https://github.com/Automattic/wp-calypso/pull/50191) @yuliyan 

#### Testing instructions

1. Load the diff on your sandbox (D57237-code) and confirm that the above changes work correctly
2. When a change has been tested to work correctly, please mark it as checked in the list above
3. Test that the new version of the plugin doesn't break Atomic sites by following the instructions in the "Atomic Testing" section at PCYsg-ly5-p2.
   * [x] Tested on atomic with and without Gutenberg plugin active

### CODE REVIEW

**Comment by creativecoder** [comment]
I verified both of the following for Simple and Atomic sites

- Update Newspack Blocks to v1.21.0 (#50154)
- Remove Anchor-specific welcome guide (#50161)

**Comment by StefanNieuwenhuis** [comment]
> Focused Launch - Summary View: Fix incorrect HE support link (#50115) - @StefanNieuwenhuis 

Tested and works as expected 👍

**Comment by ciampo** [comment]
- Rebased on top of trunk to include latest changes
- Added `/packages` related changes to the PR description

**Comment by ciampo** [comment]
- #50190 and #50191 don't need to be tested in the front end — Having the `Check code style for branches (Calypso)` CI task completing successfully should be enough
- #50114 tested by making sure that the dist files produced by ETK have not changed compared to `trunk`
- tested #50182, #49067, #49930 & #49883 on simple sites

**Comment by StefanNieuwenhuis** [comment]
> Focused Launch: Only launch site if checkout is complete (#48452) @StefanNieuwenhuis @razvanpapadopol

Tested and the changes look good. It works as expected.

**Comment by StefanNieuwenhuis** [comment]
> Move page template modal into its own package (#49661) @roo2

I've tested this against production to see if there are any changes and haven't seen any changes. Since this is about moving code to a package, I assume that this is expected, but since there weren't any testing instructions in the PR, I cannot be 100% sure.

**Comment by ciampo** [comment]
All changes included so far have been tested on simple sites.

On hold as we investigate errors on Atomic sites, potentially related to wpcom-block-editor plugin (which would be required by this version of ETK). See p1613660495322200-slack-C02DQP0FP

**Comment by ciampo** [comment]
> On hold as we investigate errors on Atomic sites

This should be fixed after merging #50237.

Rebased to include latest changes and tested #50241 on simple sites.

**Comment by andrewserong** [comment]
Rebased to include a fix for the page layout picker, I believe this release is now ready to land.

**Review by razvanpapadopol — APPROVED** [review]
Smoke tested atomic with and without Gutenberg plugin active after WBE fix from #50237
Looks ready to 🚢

**Comment by ciampo** [comment]
Rebased in order to include #50250

**Comment by ciampo** [comment]
We weren't able to test for #50250 because we didn't know how to follow the instructions, although by looking at the code changes, they are only CSS removals — should be pretty safe (cc @Addison-Stavlo)

**Comment by ciampo** [comment]
Cleanup patches:
- remove `2.16`: D57332-code
- remove `2.17`: D57334-code

**Pull request merged**
